### PR TITLE
Version 1.0.6-1

### PR DESCRIPTION
**Pull request by aphototool**
New version
Change to cmake

### CODE REVIEW

File: me.ahola.aphototoollibre.yml [hunk 1]
```diff
@@ -7,20 +7,12 @@ finish-args:
   - --share=ipc
   - --socket=fallback-x11
   - --socket=wayland
-  - --filesystem=home
+  - --filesystem=host
```
Comment (bbhtt) [inline me.ahola.aphototoollibre.yml:10]: Why was this changed to host?

Comment (aphototool) [reply me.ahola.aphototoollibre.yml:10]: By mistake
